### PR TITLE
En starting page

### LINKED ISSUE #332: Moving taxonomy trees doesn't work
Tried to move a taxonomy tree in the backend https://en.serlo.org/taxonomy/term/organize-all using the "sort" button, but resulted in an error message. Is there a different way to move taxonomy?

### LINKED ISSUE #390: Remove "Learn maths" from english homepage

### PR DESCRIPTION
**Pull request by JonasBley**
Removed the math symbol from the starting page, added participate button and changed vision text. closes #390

### CODE REVIEW

**Comment by arekkas** [comment]
Awesome!

File: src/module/Ui/templates/layout/en/serlo-home.phtml [hunk 1]
```diff
@@ -80,68 +80,45 @@ echo $this->doctype();
     <div id="page">
         <section class="clearfix">
             <div id="content-layout" class="clearfix">
-                <?php echo $this->partial('layout/partials/notifications'); ?>
```
Comment (arekkas) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: Why did you remove this?

File: src/module/Ui/templates/layout/en/serlo-home.phtml [hunk 2]
```diff
@@ -80,68 +80,45 @@ echo $this->doctype();
...
-                    <div class="home-column">
-                        <a href="/23591">Learn maths <i class="fa fa-arrow-circle-right"></i></a><br/>
-                    </div>
+                <div class="flasher">
                 </div>
 
                 <div class="home-serlo-row home-row">
                     <div class="home-column">
                         <div class="logo-wrapper">
-                            <?php echo $this->brand()->getLogo(); ?>
```
Comment (arekkas) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: Why did you remove this? Isn't this equivalent to `<span class="serlo-logo">V</span> <div class="serlo-brand">Serlo</div> </div>`?

File: src/module/Ui/templates/layout/en/serlo-home.phtml [hunk 3]
```diff
@@ -80,68 +80,45 @@ echo $this->doctype();
...
-                        <p>
-                            Our vision is that each and every one has free access to high-quality education and can
-                            learn according to their potential - let’s get started. Together.
-                        </p>
-                        <a href="/23727">Learn more <i class="fa fa-arrow-circle-right"></i></a>
+                        <span style="padding: 15px; font-size: 16pt;"><a href="/23727">Learn more <i class="fa fa-arrow-circle-right"></i></a></span>
+                        <span style="padding: 15px; font-size: 16pt;"><a href="/23727">Participate <i class="fa fa-arrow-circle-right"></i></a></span>
                     </div>
                 </div>
-
```
Comment (arekkas) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: Blank lines help in keeping readability, please add them again :)

**Comment by arekkas** [comment]
Comments inline

**Comment by arekkas** [comment]
Could you additionally create a screenshot, so I get a better feeling of what this looks like?

**Comment by JonasBley** [comment]
screenshot 2016-09-07 17 15 04

**Comment by arekkas** [comment]
What does it look like when you're logged in?

**Comment by arekkas** [comment]
The default login is `admin@localhost` with password `123456` iirc

**Comment by JonasBley** [comment]
I have  changed things according to what you said. The screenshot is also not up to date, will create a new pull request tomorrow

**Comment by JonasBley** [comment]
Changed a small thing in the styles and need to update them on my machine as well, I will see tomorrow how it looks

**Comment by inyono** [comment]
> [...] will create a new pull request tomorrow

Not needed, just push additional commits to the same branch.

**Comment by JonasBley** [comment]
screenshot 2016-09-08 13 35 41

File: src/assets/source/styles/partials/_serlo_home.scss [hunk 4]
```diff
@@ -70,6 +56,20 @@ $homeBreakpoint-lg-max: ($homeBreakpoint-lg-min - 1);
   }
 }
 
+.home-emph-link {
+      font-size: 23px;
```
Comment (arekkas) [inline src/assets/source/styles/partials/_serlo_home.scss]: please use the standard identation, you can autoformat in most IDEs :)

File: src/module/Ui/templates/layout/en/serlo-home.phtml (hunk 1 above)
Comment (arekkas) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: @JonasBley it would be great if you could answer this :)

Comment (JonasBley) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: I think I added it again in the latest version

Comment (arekkas) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: No, it's still missing :)

Comment (JonasBley) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: K second

File: src/module/Ui/templates/layout/en/serlo-home.phtml [hunk 5]
```diff
@@ -80,68 +80,45 @@ echo $this->doctype();
...
+                <div class="flasher">
                 </div>
 
                 <div class="home-serlo-row home-row">
                     <div class="home-column">
                         <div class="logo-wrapper">
-                            <?php echo $this->brand()->getLogo(); ?>
-                            <?php echo $this->brand()->getBrand(); ?>
-                        </div>
+                            <span class="serlo-logo">V</span> <div class="serlo-brand">Serlo</div> </div>
```
Comment (arekkas) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: Could you put the trailing div into a new line?

```
                            <span class="serlo-logo">V</span> <div class="serlo-brand">Serlo</div>
                        </div>
```

**Comment by arekkas** [comment]
It would be great if you could make a screenshot of the german landing page as well :)

File: src/module/Ui/templates/layout/en/serlo-home.phtml (hunk 5 above)
Comment (JonasBley) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: Oh that part was a bit messed up, sorry. Should be better now though I cant see how stuff looks right now, it says it rsyncs the changes, but they don't show up on my machine...

Comment (arekkas) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: If that happens, kill the rsync process and restart it with `vagrant rsync-auto`

**Comment by JonasBley** [comment]
I didnt change anything on the German landing page

File: src/module/Ui/templates/layout/en/serlo-home.phtml (hunk 5 above)
Comment (JonasBley) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: I tried that, I'll try again

Comment (JonasBley) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: rsync-auto is integrated in the vagrant up command or not?

**Comment by arekkas** [comment]
> I didnt change anything on the German landing page

You did, this PR contains 3 changed files, on of them is the german landing page: https://github.com/JonasBley/athene2/blob/f8ffafc1175e084d10ff7c06531c79a135e873ef/src/module/Ui/templates/layout/de/serlo-home.phtml

**Comment by arekkas** [comment]
You can view all changes of your PR on GitHub:

sdfgsgdf

**Comment by JonasBley** [comment]
I created the class "home-emph-link" and moved the link properties from the class "home-column" to the new class, so I added all the links on the German page to the home-emph-link class. 
I needed to do that, because I couldnt otherwise change the participate and the about button on en.serlo.org

**Comment by JonasBley** [comment]
I didnt actually change anything on the German page, it looks just the same

**Comment by arekkas** [comment]
> I didnt actually change anything on the German page, it looks just the same

Haha, my bad, got it :)

File: src/module/Ui/templates/layout/de/serlo-home.phtml [hunk 6]
```diff
@@ -184,16 +184,16 @@ echo $this->doctype();
...
                             </p>
-                            <a href="/21565" class="home-hide-md">Spenden <i class="fa fa-arrow-circle-right"></i></a>
+                            <a href="/21565" class="home-hide-md" class="home-emph-link">Spenden <i class="fa fa-arrow-circle-right"></i></a>
                         </div>
                     </div>
 
                     <div class="home-community-link-row home-row">
                         <div class="home-column home-col-md-3">
-                            <a href="/19869" class="home-show-md">Mitmachen <i class="fa fa-arrow-circle-right"></i></a>
+                            <a href="/19869" class="home-emph-link" class="home-show-md">Mitmachen <i class="fa fa-arrow-circle-right"></i></a>
```
Comment (arekkas) [inline src/module/Ui/templates/layout/de/serlo-home.phtml]: This should be `class="home-emph-link home-show-md"`

File: src/module/Ui/templates/layout/de/serlo-home.phtml [hunk 7]
```diff
@@ -184,16 +184,16 @@ echo $this->doctype();
...
                         </div>
                         <div class="home-column home-col-md-3">
-                            <a href="/21565" class="home-show-md">Spenden <i class="fa fa-arrow-circle-right"></i></a>
+                            <a href="/21565" class="home-emph-link" class="home-show-md">Spenden <i class="fa fa-arrow-circle-right"></i></a>
```
Comment (arekkas) [inline src/module/Ui/templates/layout/de/serlo-home.phtml]: This should be `class="home-emph-link home-show-md"`

File: src/module/Ui/templates/layout/en/serlo-home.phtml [hunk 8]
```diff
@@ -143,11 +151,13 @@ echo $this->doctype();
                     </div>
 
                     <div class="home-community-link-row home-row">
+
                         <div class="home-column home-col-md-3">
-                            <a href="/27469" class="home-show-md">Participate <i class="fa fa-arrow-circle-right"></i></a>
+                            <a href="/27469" class="home-emph-link" class="home-show-md">Participate <i class="fa fa-arrow-circle-right"></i></a>
```
Comment (arekkas) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: This should be `class="home-emph-link home-show-md"`

File: src/module/Ui/templates/layout/en/serlo-home.phtml [hunk 9]
```diff
@@ -143,11 +151,13 @@ echo $this->doctype();
...
                         </div>
+
                         <div class="home-column home-col-md-3">
-                            <a href="/39837" class="home-show-md">Donate <i class="fa fa-arrow-circle-right"></i></a>
+                            <a href="/39837" class="home-emph-link" class="home-show-md">Donate <i class="fa fa-arrow-circle-right"></i></a>
```
Comment (arekkas) [inline src/module/Ui/templates/layout/en/serlo-home.phtml]: This should be `class="home-emph-link home-show-md"`

**Comment by JonasBley** [comment]
screenshot 2016-09-08 19 22 04

**Comment by arekkas** [comment]
Awesome! Thank you!
